content/en/database_monitoring/connect_dbm_and_apm.md

```diff
@@ -72,7 +73,7 @@ APM tracer integrations support a *Propagation Mode*, which controls the amount
 
 \* [CommandType.StoredProcedure][25] not supported
 
-\*\* Full mode SQL Server/Java:
+\*\* Full mode SQL Server for Java/.NET:
```

Comment:
Can we indent the bulleted list that follows this sentence?

Reply:
Yep indented it thanks!